projects/packages/search/src/dashboard/components/record-meter/notice-box.jsx

```diff
@@ -101,18 +99,13 @@ export function NoticeBox( props ) {
 		! dismissedNoticesString.includes( NO_INDEXABLE_ITEMS ) &&
 		activeNoticeIds.push( NO_INDEXABLE_ITEMS );
 
-	// check if over limit
-	typeof props.tierMaximumRecords === 'number' &&
-		props.recordCount > props.tierMaximumRecords &&
-		! dismissedNoticesString.includes( OVER_RECORD_LIMIT ) &&
-		activeNoticeIds.push( OVER_RECORD_LIMIT );
-
 	// check if close to reaching limit
 	typeof props.tierMaximumRecords === 'number' &&
 		props.recordCount > props.tierMaximumRecords * CLOSE_TO_LIMIT_PERCENT &&
 		props.recordCount < props.tierMaximumRecords &&
 		! dismissedNoticesString.includes( CLOSE_TO_LIMIT ) &&
 		activeNoticeIds.push( CLOSE_TO_LIMIT );
+	activeNoticeIds.push( CLOSE_TO_LIMIT );
```

Comment:
Is this addition intentional? Wouldn't this line lead to always showing the near-the-limit notice?

Reply:
Excellent spot! This is left over from my testing. I'll remove it 👍